pom.xml

```diff
@@ -49,7 +49,7 @@
   </developers>
 
   <properties>
-    <maven.compiler.release>8</maven.compiler.release>
+    <maven.compiler.release>11</maven.compiler.release>
```

Comment:
i'm worried that it won't work on the CI. For now, we don't have access to the source code of the CI, so we can't tell if there is jdk 11 available. I suggest we separate the two topics, and we get back to upgrading to java 11 once we have access to the CI source code

Reply:
reverted